config-ui/src/pages/blueprints/index.jsx

```diff
@@ -89,6 +90,14 @@ const Blueprints = (props) => {
     detectPipelineProviders
   } = usePipelineManager()
 
+  const {
+    pagedData,
+    setFilterParams,
+    setFilterFunc,
+    setData,
```

Comment:
For less ambiguity surrounding the context of the data that is being set, we can alias this to `setPaginatorData`.

```suggestion
    setData: setPaginatorData,
```
Afterwards, update the associated effect call on `Ln 284` to use `setPaginatorData`

Reply:
setDataWithDefault